### PR TITLE
Update _lockup--option-a.scss

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW 

# Summary
- Updates line height on lockup option a line one to support multiple lines.

# Needed By (Date)
- Whenever

# Urgency
- Low

# Steps to Test

1. Create a really long title in the lockup line 1 on option A and review at different screen sizes.

# Affected Projects or Products
- D8CORE
- This project

# Associated Issues and/or People
- D8CORE-2562
- https://github.com/SU-SWS/stanford_basic/pull/177

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: core/src/scss/components/lockup/_lockup--option-a.scss [hunk 1]
```diff
@@ -33,7 +33,8 @@
 
     @include grid-media('md') {
       font-size: 3.2rem;
-      line-height: 0.7em;
+      line-height: 1.15em;
+      @include margin(-8px null -6px null);
```
Comment (yvonnetangsu) [inline core/src/scss/components/lockup/_lockup--option-a.scss:37]: Hey so I know where this is coming from (a certain site that was launched this morning), is there a reason why they don't use option-m for this instead? Totally understand if it's a D8/Drupal thing or that option-m isn't available right now.

Screen Shot 2020-08-24 at 5 03 11 PM

**Comment by sherakama** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr722-vv0brga6sy9mspjcupjoi0e4eynmfk9n.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f445271353bf755280f1874

File: core/src/scss/components/lockup/_lockup--option-a.scss (hunk 1 above)
Comment (sherakama) [reply core/src/scss/components/lockup/_lockup--option-a.scss:37]: Great question. I'll ask.
https://stanfordwebservices.slack.com/archives/GJQJ7HTK7/p1598373210028400

Comment (sherakama) [reply core/src/scss/components/lockup/_lockup--option-a.scss:37]: We're going to add lockup option M to the list in Drupal and adjust the default line height for option A in stanford_basic.

Comment (yvonnetangsu) [reply core/src/scss/components/lockup/_lockup--option-a.scss:37]: > Great question. I'll ask.
> https://stanfordwebservices.slack.com/archives/GJQJ7HTK7/p1598373210028400

I can't seem to access this Slack channel. Let me know if I should be on it 😃

Comment (sherakama) [reply core/src/scss/components/lockup/_lockup--option-a.scss:37]: You have been added.

**Pull request closed without merging**